site/docs/install.md

```diff
@@ -12,4 +12,14 @@ See the instructions for installing Bazel on:
 *   [macOS](install-os-x.md)
 *   [Windows](install-windows.md)
 
+## Community Supported Platforms
```

Comment:
"Community Supported Packages"?

Reply:
Also, "Community-Supported".

Reply:
Also, "Community-Supported".

Reply:
Sorry, platforms is not the right word. I've fixed it to `Community-Supported Packages`